### ISSUE TITLE
The ontology (kind) within the integrals tab ( signal, solvent, reference etc.) is confusing and not defined. The kind of the signal should be annotated within the peaks sections.

### ISSUE THREAD

**Issue by hamed-musallam** [cheminfo/nmrium#1836]

**Comment by lpatiny** [comment]
Yes indeed. We could hide this column by default for now. The idea is that only signals are part of integration and the other ones are currently ignored in the sum of the integral.

What are the proposals ? Just documentation ?

In the future we would like to have multiple set of integrations depending if it is P1, P2, P3

**Comment by CS76** [comment]
I am not sure if there is a proposal to use a rich select component, where we could provide hints inline for the options. Hover ever as a quick solution; we can give title attribute to option tags. So, upon hover, a hint is shown of what the value represents.

https://user-images.githubusercontent.com/4065285/205150140-90fa6c0a-c888-40a4-b160-fdb7d3c0579b.mov
